spec/services/get_share_price_service_spec.rb

```diff
@@ -0,0 +1,20 @@
+require 'rails_helper'
+
+RSpec.describe GetSharePriceService, type: :service do
+  let(:share) { 'petz3' }
+  let(:invalid_share) { 'invalid' }
+
+  describe 'get share price' do
+    context 'when invoking the service' do
+      it 'returns share price' do
+        price = GetSharePriceService.new(share).share_price
+        expect(price).to eq(1866.0)
+      end
+
+      it 'return an error 'do
+        price = GetSharePriceService.new(invalid_share).share_price
+        expect { price }.to rescue(StandardError)
```

Comment:
é melhor usar o `raise_error` aqui, pra seguir o padrão do rspec: https://relishapp.com/rspec/rspec-expectations/docs/built-in-matchers/raise-error-matcher